consent-ws/src/main/java/org/broadinstitute/consent/http/service/DataAccessParser.java

```diff
@@ -194,6 +195,43 @@ public PDAcroForm fillDARForm(Document dar, Map<String, String> researcherProper
                     field.setValue(getYesOrNoValue(dar.getBoolean(DarConstants.POA)));
                     break;
                 }
+                case DarConstants.CHECK_COLLABORATOR: {
+                    field.setValue(getYesOrNoValue(dar.getBoolean(DarConstants.CHECK_COLLABORATOR)));
+                    break;
+                }
+                case "userStatus": {
+                    field.setValue(getDefaultValue(StringUtils.capitalize(role.getStatus())));
+                    break;
+                }
+                case "adminComment": {
+                    field.setValue(getDefaultValue(StringUtils.capitalize(role.getRationale())));
```

Comment:
Just double checking that `role.getRationale()` is really the `adminComment` case.

Reply:
yes, rational and comments are saved in the same place